src/main/java/sixgaezzang/sidepeek/skill/controller/SkillController.java

```diff
@@ -0,0 +1,28 @@
+package sixgaezzang.sidepeek.skill.controller;
+
+import jakarta.validation.constraints.Size;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import sixgaezzang.sidepeek.skill.dto.SkillSearchResponse;
+import sixgaezzang.sidepeek.skill.serivce.SkillService;
+
+@RestController
+@RequestMapping("/skills")
```

Comment:
```suggestion
@RequestMapping("/api/v1/skills")
```
저희 URI에 `/api/v1/` 붙여주기로 하지 않았나요..? 노션에 기재된 API 명세서와 다른 것 같습니다!

Reply:
앗 요거는 yml에서 전역으로 설정해준 것으로 알고 있어서 여기서는 붙이지 않았습니다!
나중에 main 쪽에 application.yml 확인해주시면 감사하겠습니당!!